index.html

```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
+
```

Comment:
![pr-title](https://user-images.githubusercontent.com/93039359/185356548-4af2912b-ad00-4623-a7fb-60e05ef880e8.png)
**[Optional]**
- [ ] Your PR has a title that is not so descriptive enough. Because **add 2 sections** is not describing what are you doing in the project. PR title fully describes what is happening in this branch. You could use **Portfolio: finish mobile version** for your PR title. :+1: